<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>readBook</class>
 <widget class="QDialog" name="readBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Read a book</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 15pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Iva Raykova</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(116, 118, 129);
font: 300 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Writer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;
border-radius: 4px;
background-color: #FFF;
</string>
   </property>
   <property name="text">
    <string>My Books</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>1171</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FFF;</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1169</width>
      <height>639</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-70</y>
       <width>1171</width>
       <height>631</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ffffff;</string>
     </property>
     <property name="text">
      <string>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla ac tincidunt ante, sed blandit erat. Curabitur nec blandit erat, vitae tempor mauris. Sed tristique molestie leo, ac facilisis libero scelerisque nec. Proin imperdiet odio eget nibh faucibus, at cursus lacus rutrum. Pellentesque ornare pharetra lacus, eu pulvinar orci. Fusce a libero massa. Ut felis mauris, sollicitudin a cursus eget, porta vel eros. Sed at efficitur turpis.

Nunc efficitur luctus metus ac luctus. Sed viverra vel urna nec facilisis. Integer tempor eget dolor id fermentum. Aenean sed libero eget ex volutpat maximus. Fusce odio dolor, ultrices in ultricies eget, tincidunt eget mauris. Praesent vehicula diam in lacus porta molestie. Etiam eget finibus eros, quis lobortis sapien. In hac habitasse platea dictumst. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Pellentesque euismod, arcu nec mattis ultricies, magna risus aliquet nisl, nec sodales odio orci vitae neque. Duis bibendum nisl sit amet aliquet posuere. Pellentesque eu ipsum sit amet augue faucibus aliquam et at nibh. Aenean non justo ac massa cursus finibus lacinia vel est. Lorem ipsum dolor sit amet, consectetur adipiscing elit.

Aliquam dolor nibh, feugiat eu ipsum ac, maximus consequat mauris. Integer sit amet pretium odio, eu elementum eros. Donec et orci consectetur, mollis dolor sit amet, imperdiet ante. Maecenas pellentesque facilisis dolor nec cursus. Cras ac hendrerit metus. Sed sollicitudin, purus nec tincidunt eleifend, tellus orci elementum orci, et euismod arcu augue non nibh. Aliquam imperdiet pretium elit, ut hendrerit leo interdum eget. Ut eget felis dignissim metus faucibus accumsan vitae id arcu. Phasellus auctor pulvinar arcu vel sollicitudin. Vivamus rutrum mattis ex, a pharetra purus.

Praesent aliquet, lorem vitae tincidunt mattis, arcu nunc dictum sapien, sit amet interdum leo mi ut dui. Sed ultricies vehicula nisl, eu aliquet eros semper id. Aliquam et neque at diam suscipit maximus at quis mi. Proin sit amet scelerisque eros. Nullam tincidunt lorem et velit ullamcorper, sed vulputate neque consectetur. Ut imperdiet efficitur justo. Ut vel tempus erat. Proin metus lacus, varius eget suscipit ac, interdum non tortor.

Donec sit amet venenatis urna. Quisque vehicula tempus nibh sit amet euismod. Donec vehicula arcu et turpis vehicula fringilla. Praesent fringilla metus id orci tincidunt finibus. Curabitur lacinia id libero id volutpat. Sed sit amet porttitor ligula. Mauris vel pharetra orci.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>172</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 17pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Lorem Ipsum</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Test Test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
